sdk/communication/communication-sms/recordings/node/smsclient_playbackrecord_aad_when_sending_sms/recording_throws_an_exception_when_sending_from_an_invalid_number.js

```diff
@@ -5,7 +5,78 @@ module.exports.hash = "e40572f5ed6d1f0cd7c4a5809ef38226";
 module.exports.testInfo = {"uniqueName":{},"newDate":{}}
 
 nock('https://endpoint', {"encodedQueryParams":true})
-  .post('/SomeTenantId/oauth2/v2.0/token', "response_type=token&grant_type=client_credentials&client_id=SomeClientId&client_secret=SomeClientSecret&scope=https%3A%2F%2Fcommunication.azure.com%2F%2F.default")
+  .get('/common/discovery/instance')
+  .query(true)
+  .reply(200, {"tenant_discovery_endpoint":"https://login.microsoftonline.com/SomeTenantId/v2.0/.well-known/openid-configuration","api-version":"1.1","metadata":[{"preferred_network":"login.microsoftonline.com","preferred_cache":"login.windows.net","aliases":["login.microsoftonline.com","login.windows.net","login.microsoft.com","sts.windows.net"]},{"preferred_network":"login.partner.microsoftonline.cn","preferred_cache":"login.partner.microsoftonline.cn","aliases":["login.partner.microsoftonline.cn","login.chinacloudapi.cn"]},{"preferred_network":"login.microsoftonline.de","preferred_cache":"login.microsoftonline.de","aliases":["login.microsoftonline.de"]},{"preferred_network":"login.microsoftonline.us","preferred_cache":"login.microsoftonline.us","aliases":["login.microsoftonline.us","login.usgovcloudapi.net"]},{"preferred_network":"login-us.microsoftonline.com","preferred_cache":"login-us.microsoftonline.com","aliases":["login-us.microsoftonline.com"]}]}, [
+  'Cache-Control',
+  'max-age=86400, private',
+  'Content-Length',
+  '980',
+  'Content-Type',
+  'application/json; charset=utf-8',
+  'Strict-Transport-Security',
+  'max-age=31536000; includeSubDomains',
+  'X-Content-Type-Options',
+  'nosniff',
+  'Access-Control-Allow-Origin',
+  '*',
+  'Access-Control-Allow-Methods',
+  'GET, OPTIONS',
+  'P3P',
+  'CP="DSP CUR OTPi IND OTRi ONL FIN"',
+  'x-ms-request-id',
+  '79ed6191-897d-4dfd-924d-9ec23e320c00',
+  'x-ms-ests-server',
+  '2.1.11722.21 - WUS2 ProdSlices',
+  'Set-Cookie',
+  'fpc=An8aJvjkv89FlfmGvJw55UyMQo4QBQAAABH4QdgOAAAA; expires=Sat, 26-Jun-2021 20:29:10 GMT; path=/; secure; HttpOnly; SameSite=None',
+  'Set-Cookie',
+  'esctx=AQABAAAAAAD--DLA3VO7QrddgJg7WevrN3-sT-lkX_Iew10TwlmO77QOWObJg3FOTznlKFp-aJQX7Wv9BG7qr6XK9B-r5o7vpJ2_Fe-LJazF7VYQRS24T-k3cPoMRFb40n_iusHR3O8Wv1lUs1UgN4PGLlIEq8nfgZhBR2VoTVxBrh1aSpESn_tvABaqdfSzn1JzJDYgXZMgAA; domain=.login.microsoftonline.com; path=/; secure; HttpOnly; SameSite=None',
```

Comment:
I don't know what `fpc` and `esctx` cookies contain, do we need to sanitize them?

Reply:
text analytics didn't sanitize it in their PR but i'll follow up!
https://github.com/Azure/azure-sdk-for-js/pull/15056/files

Reply:
I’m aware of this, but we’re still investigating if this represents a security risk.

Reply:
@DominikMe I got in contact with the team that develops this feature and they said these cookies are not a security issue. However, they suggested us to remove all of these cookies in case the service decides to introduce one that may expose sensitive information in the future.

cc: @HarshaNalluru , @KarishmaGhiya